tests/resource/test_resource_getters.py

```diff
@@ -60,6 +60,9 @@ def test_resource_admin_get_universal_access_raises_an_exception_when_extract_fa
         with self.assertRaises(SecurityRequestError):
             self.resource_admin.get_universal_access("TESTING", "ELIJTEST")
 
+    # ============================================================================
+    # My Access
+    # ============================================================================
     def test_resource_admin_get_my_access_returns_valid_when_read(
```

Comment:
I think the following naming convention is cleaner here:

```python
def test_resource_admin_get_my_access_read(self):
def test_resource_admin_get_my_access_none(self):
def test_resource_admin_get_my_access_raises_an_exception_when_extract_fails(self):
def test_resource_admin_get_audit_rules(self):
def test_resource_admin_get_audit_rules_raises_an_exception_when_extract_fails(self):
```

Reply:
agree this is better. change in next commit.